tests/sentry/api/endpoints/test_group_integration_details.py

```diff
@@ -156,7 +156,12 @@ def test_get_feature_disabled(self):
 
         path = f"/api/0/issues/{self.group.id}/integrations/{integration.id}/?action=create"
 
-        with self.feature({"organizations:integrations-issue-basic": False}):
+        with self.feature(
+            {
+                "organizations:integrations-issue-basic": False,
+                "organizations:integrations-issue-sync": False,
+            }
+        ):
```

Comment:
Cases such as this one, where the test calls for more than one flag to be adjusted, make me feel better about investing in the change. Previously, you could have actually gotten away without declaring _either_ as false; `with self.feature({})` would have sufficed.